crates/fluvio-connector-common/src/producer.rs

```diff
@@ -0,0 +1,41 @@
+use fluvio::{FluvioConfig, Fluvio, TopicProducer, TopicProducerConfigBuilder};
+use crate::{ConnectorConfig, Result};
+
+use crate::{ensure_topic_exists, smartmodule::smartmodule_chain_from_config};
+
+pub async fn producer_from_config(config: &ConnectorConfig) -> Result<(Fluvio, TopicProducer)> {
```

Comment:
What do you think about instead of returning a Fluvio instance, accepting a Fluvio instance so we can reuse it if we want to create a consumer or maybe other structs?

Reply:
Fluvio instance should be created using the info from ConnectorConfig. Right now, it is only `client_id,` but potentially, it can be more.  Here we control this process. 

I don't anticipate there will be connectors that will need to create both producers and consumers.

Reply:
But if there will be such connectors, we will create another methods convenient for them